tests/test_valitorpay.py

```diff
@@ -33,11 +33,22 @@ def credentials():
         'apikey': apikey,
     }
 
-
 @pytest.fixture
 def valitor(credentials):
     return valitor_python.ValitorPayClient(**credentials)
 
+
+
+@pytest.fixture
+def wrong_credentials():
+    return {
+        'apikey': "ApiKeyUAT.RxE3xfVnksfC/UnoDpu31UQ/gY"
```

Comment:
Let's change the "wrong" value to something obviously made up, e.g. "DummyApiKey"